packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -1506,4 +1540,52 @@ describe.each([
         expectQuery({ equal: { "1:1:name": "none" } }).toFindNothing())
     })
   })
+
+  // This will never work for Lucene.
+  // TODO(samwho): fix for SQS
```

Comment:
Is this meant to be done before merging?

Reply:
Ah, @mike12345567 actually fixed this. Lemme "uncomment" the test. Good catch!

Reply:
Was just the comment that needed removing, the condition was already allowing the SQS test variant to run.